servers/feature-flags/src/graphql/resolvers.ts

```diff
@@ -60,28 +60,20 @@ const convertToUnleashAssignment = (
   context: UnleashContext,
 ): UnleashAssignment[] => {
   return toggles.map((toggle: FeatureInterface) => {
-    // Unleash in a recent update returns null or an empty array for variants in the API response
-    // We need to check for both
-    if (
-      !toggle.variants ||
-      (toggle.variants instanceof Array && toggle.variants.length === 0)
-    ) {
+    const variant = instance.forceGetVariant(toggle.name, context);
+
+    if (!variant || variant.name == 'disabled') {
```

Comment:
I'm a little confused about this. can you explain why 'disabled' name is included in this?

Reply:
I guess if it's not making an external request for isEnabled then there's not really an issue -- would just return false

Reply:
@kschelonka yea.. its how unleash is doing it now. https://docs.getunleash.io/reference/strategy-variants#the-disabled-variant

Basically, if your flag is turned on, but has no variants, the variant you get is "disabled". In this scernario, we just return whether or not the flag is on for a user.